src/node/waku_node.py

```diff
@@ -84,6 +109,46 @@ def start(self, **kwargs):
             logger.error(f"REST service did not become ready in time: {ex}")
             raise
 
+    @retry(stop=stop_after_delay(5), wait=wait_fixed(0.1), reraise=True)
+    def register_rln(self, **kwargs):
+        logger.debug("Registering RLN credentials...")
+        self._docker_manager.create_network()
+        self._ext_ip = self._docker_manager.generate_random_ext_ip()
```

Comment:
do we need all those flags for the registation ?
in the [nwaku compose](https://github.com/waku-org/nwaku-compose/blob/master/register_rln.sh#L23-L27) repo I don't see them
let's keep only what's needed

Reply:
All flags may change infrequently. I would still keep 2 of them in outside in environment  
"rln-relay-eth-client-address" and  "rln-relay-eth-contract-address".  rln-relay-cred-password we can probably hardcode in env_vars.py. Private keys are completely under our control, so could be hardcoded two, the only catch is someone could dry out our balance if we disclose them. Still better to keep them out of code.

Reply:
I was not completely sure for their purpose. I will go through them and remove.

Reply:
Fixed at https://github.com/waku-org/waku-interop-tests/pull/16/commits/67d0666e8f9a7348a1b304dafaac568c70565ed8